src/retrace-server-cleanup

```diff
@@ -93,7 +93,7 @@ if __name__ == "__main__":
                     continue
 
                 task.create_worker().clean_task()
-                task.set_log("Killed by garbage collector\n", True)
+                task.set_log("Task was killed due to running for too long or taking too many resources.\n", True)
```

Comment:
I'm not the best English speaker, but the for in "running for too long" sounds weird to me. Are you sure it should be there?